README.md

```diff
@@ -1,3 +1,20 @@
 # fritz
 
 Frontend for Reviewing Interesting Transients from ZTF-II
+
+## Bare-bones end-to-end demo
+
+Clone the repo from `GitHub`, pull the submodules (`skyportal` and `kowalski`), and make the `fritz` utility executable:
```

Comment:
The Fritz utility should check and initialize submodules.

The Fritz utility should also be executable by default in the repository---is it not for you?

(I.e., this step of the instructions, apart from the git clone, should happen implicitly.)

Reply:
Added submodule init, see [here](https://github.com/fritz-marshal/fritz/pull/4/commits/aa65efb83003953a612fdcb11bc985c4637a3365)

The utility is executable, fixed the instructions.